### ISSUE TITLE
Pacman demo not working - training loss never goes below 1.3 

### ISSUE THREAD

**Issue by Guitlle** [tensorflow/tfjs#599]
When using this demo https://storage.googleapis.com/tfjs-examples/webcam-transfer-learning/dist/index.html  I attempt to reproduce what is seen in this video https://www.youtube.com/watch?v=YB-kfeNIPCE 
However, loss never gets small. It is stuck at 1.3 unlike the demo in the video, where it gets down to 0.008 or so. The settings are the same as in the video and I attempted to get a clean background but had no success with reproducing that result. 

Is the demo in the video different from the one in the tfjs examples? 

Why loss never get smaller? I even tried adding more hidden layers and 200 epochs but had no success.

**Comment by bileschi** [comment]
It worked for me right now, is it possible there is some incompatibility with the camera?   Were you sure to select representative training images for all 4 directions?


![image](https://user-images.githubusercontent.com/547150/44037166-a024f648-9ee1-11e8-81b7-668bf9c760f3.png)

**Comment by nsthorat** [comment]
@Guitlle which device were you running it on?

**Comment by Guitlle** [comment]
Hi. I used a HP Envy with some integrated 3d acceleration device. I used ubuntu and chrome. The demo did spend a while doing processing but it could not optimize the model.
